packages/reqresp/test/utils/index.ts

```diff
@@ -21,14 +21,24 @@ export async function* arrToSource<T>(arr: T[]): AsyncGenerator<T> {
  * Wrapper for type-safety to ensure and array of Buffers is equal with a diff in hex
  */
 export function expectEqualByteChunks(chunks: Uint8Array[], expectedChunks: Uint8Array[], message?: string): void {
-  expect(chunks.map(toHexString).join("").replace(/0x/g, "")).to.deep.equal(
-    expectedChunks.map(toHexString).join("").replace(/0x/g, ""),
-    message
-  );
+  if (message) {
+    expect(chunks.map(toHexString).join("").replace(/0x/g, "")).toEqualWithMessage(
+      expectedChunks.map(toHexString).join("").replace(/0x/g, ""),
+      message
+    );
+  } else {
+    expect(chunks.map(toHexString).join("").replace(/0x/g, "")).toEqual(
+      expectedChunks.map(toHexString).join("").replace(/0x/g, "")
+    );
+  }
```

Comment:
Its interesting that this test case, and the one below, changed

Reply:
It's just a condition to choose between toEqual and toEqualWithMessage. We can't pass undefined as message to the later assertion.